hack/bin/deps.bzl

```diff
@@ -268,52 +268,48 @@ filegroup(
 def install_kubetest2():
     # install kubetest2 binary
     http_file(
-       name = "kubetest2_darwin",
-       executable = 1,
-       sha256 = "5b20aadd05eca47dead180a7c8296d75e81c184aabf182d4a41ef96597db543d",
-       urls = ["https://storage.googleapis.com/crdb-bazel-artifacts/osx/kubetest2"],
+        name = "kubetest2_darwin",
```

Comment:
Can you include the updated sha256? Bazel _should_ have a warning log line when you run the build with the SHA256 telling you that it'll make the build more reproducible.

Reply:
Done